blockware/fireworks/src/main.cpp

```diff
@@ -187,17 +295,19 @@ void tick()
   // redraw entire canvas on every tick
 
   // base
+  updateWakeState();
+  drawParticles();
```

Comment:
oh nice this is why the logo is above the particles haha

Reply:
Yes, I thought it looked better that way. :)